src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_SymbolSpec.cs

```diff
@@ -397,9 +397,6 @@ private static string ToEditorConfigString(this SymbolKindOrTypeKind symbol)
                 case TypeKind.Delegate:
                     return "delegate";
 
-                case TypeKind.Module:
-                    return "module";
```

Comment:
💭 Since `module` was already an option, should we make `module` apply to both classes and modules, while `module` applies to just modules?

Reply:
@sharwell The existing code for module didn't have any user visible effect per my understanding. The parser doesn't parse "module" (ie, the code can go from symbol kind to editorconfig string, but not the other way). So this really had no effect other than preventing Visual Studio from crashing when `ToEditorConfigString` is called with TypeKind.Module, and that's why I modified `SymbolSpecificationViewModel` so that Visual Studio doesn't call this method with module kind.

But I'd love if you take another look and confirm if I'm correct.

Reply:
It might have only been usable from Tools &rarr; Options.

Reply:
@sharwell Hmm... It didn't work from Tools → Options for me in 17.3 Preview 2, but it works in VS 2019 (16.11.3). I'm likely hitting https://github.com/dotnet/roslyn/issues/61844.

I'll follow https://github.com/dotnet/roslyn/pull/62141#discussion_r907700360